packages/@react-aria/grid/src/useGrid.ts

```diff
@@ -53,7 +53,12 @@ export interface GridProps extends DOMProps, AriaLabelingProps {
   /** Handler that is called when a user performs an action on the row. */
   onRowAction?: (key: Key) => void,
   /** Handler that is called when a user performs an action on the cell. */
-  onCellAction?: (key: Key) => void
+  onCellAction?: (key: Key) => void,
+  /**
+   * Whether the grid allows the user to clear all selected items via Escape.
+   * @default false
+   */
+  disallowClearAll?: boolean
```

Comment:
ok naming... 😂 

* Should have selection in the name? like `disallowClearSelection`? but that's also not right, you can deselect individual items and end up with nothing selected.
* Should it have something specifically about Escape in the name? `disallowEscape`? `shouldDisableEscape`? * More explicit: `shouldEscapeClearSelection` but then the default is `true`?
* Non-boolean: `escapeKeyBehavior = 'clearSelection' | 'none'`?

Reply:
I think i kinda like the non-boolean the most, it's most informative

Reply:
I'm a fan of the non-boolean one too.